plugins/web/src/server_command.cpp

```diff
@@ -164,7 +164,7 @@ request_dispatcher_actor::behavior_type request_dispatcher(
 
 void setup_route(caf::scoped_actor& self, std::unique_ptr<router_t>& router,
                  std::string_view prefix, request_dispatcher_actor dispatcher,
-                 vast::rest_endpoint endpoint,
+                 const server_config& config, vast::rest_endpoint endpoint,
```

Comment:
Could you adapt the example config with a commented illustration how to use this?

Reply:
Right now there's nothing exposed to the user, we just always allow arbitrary origins in dev mode.